pwndbg/commands/vmmap.py

```diff
@@ -366,3 +366,23 @@ def vmmap_load(filename) -> None:
     for page in pages:
         pwndbg.gdblib.vmmap.add_custom_page(page)
         print("%r added" % page)
+
+
+parser = argparse.ArgumentParser(description="Explore a page, trying to guess permissions.")
+parser.add_argument(
+    "address", type=pwndbg.commands.sloppy_gdb_parse, help="Address of the page to explore"
+)
+
+
+@pwndbg.commands.ArgparsedCommand(parser, category=CommandCategory.MEMORY)
+@pwndbg.commands.OnlyWhenRunning
+def vmmap_explore(address: int | str) -> None:
```

Comment:
We have a check for int only later on

```suggestion
def vmmap_explore(address: int) -> None:
```